### PR TITLE
#396  Check if services can start

### LINKED ISSUE #396: Check containers building and starting. stb2
Now we have the problem. We don't check if `docker build` and/or `docker up` of new containers was broken with code changes

### PR DESCRIPTION
**Pull request by duker33**
Closes #396

### CODE REVIEW

File: docker/drone_env/credentials [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+POSTGRES_PASSWORD=pass
+POSTGRES_DB=se_prod
+POSTGRES_INITDB_XLOGDIR=/var/lib/postgresql/xlog
+STAGE_DB_NAME=se_stage
+RABBITMQ_DEFAULT_USER=user
+RABBITMQ_DEFAULT_PASS=pass
+REDIS_PASSWORD=pass
+
+# Credentials for ftp with files from 1C
+FTP_PASS=e01ebbd06176f46b1f732bd0868ad6be
```
Comment (artemiy312) [inline docker/drone_env/credentials]: remove it

File: .drone.yml [hunk 2]
```diff
@@ -127,6 +127,29 @@ pipeline:
...
       event: [push, pull_request]
 
+  docker-up:
+    image: docker/compose:1.17.1
+    commands:
+      - cd docker
+      - cp drone_env/* env_files/
+      - cp drone_env/.env .
+      # 1. start containers
+      - docker-compose stop
```
Comment (artemiy312) [inline .drone.yml]: Seems it is an unnecessary line

File: docker/drone_env/.env [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+POSTGRES_PASSWORD=pass
+POSTGRES_DB=se_prod
+POSTGRES_INITDB_XLOGDIR=/var/lib/postgresql/xlog
+STAGE_DB_NAME=se_stage
+RABBITMQ_DEFAULT_USER=user
+RABBITMQ_DEFAULT_PASS=pass
+REDIS_PASSWORD=pass
+
+# Credentials for ftp with files from 1C
+FTP_PASS=e01ebbd06176f46b1f732bd0868ad6be
```
Comment (artemiy312) [inline docker/drone_env/.env]: Let's remove it

File: .drone.yml (hunk 2 above)
Comment (duker33) [reply .drone.yml]: @artemiy312 , without it we get "port is already allocated".
But seems i got more good decision to fix it

File: docker/drone_env/.env (hunk 3 above)
Comment (duker33) [reply docker/drone_env/.env]: TODO

File: docker/drone_env/credentials (hunk 1 above)
Comment (duker33) [reply docker/drone_env/credentials]: Done.

**Comment by artemiy312** [comment]
@duker33 Can we use here docker/create-env instead of hard-coded data?

**Comment by duker33** [comment]
@artemiy312 , this data is specific for drone. So, i don't know how create_env can help us here.
Anyway we should store this drone env data in cvs

**Comment by duker33** [comment]
got no answer for 24h. So, merge it

**Pull request merged**